src/telliot/utils/app.py

```diff
@@ -42,6 +42,16 @@ class LogLevel(str, enum.Enum):
     CRITICAL = "critical"
 
 
+class TelliotConfig(ConfigOptions):
+    """Shared telliot configuration object
+
+    This class maintains configuration entries shared by all
+    telliot `Applications`.  It should be updated as needed.
+    """
+
+    infura_key: str = "Insert Infura Private Key Here"
```

Comment:
Maybe change this to be wallet private key, since not everyone's going to use infura, but everyone's going to need a wallet to use Telliot. Thoughts?

Reply:
Yea, I just made this up as an example :).  Still need to think about what should go in it.
I think RPCEndpoint objects generally would be imported from config files as entire objects.

Reply:
Sounds good